app/views/layouts/_bgeigie_import_filter.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= show_filters :bgeigie_import,
+                 [:q, :uploaded_after, :uploaded_before, :status, :rejected_by, :by_user_name,:by_user_id] %>
+
+<%= filter_modal :bgeigie_imports do %>
+  <table>
+    <tbody>
+    <tr>
+      <td>
+        <b><%= t(".by_user") %></b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <%= filter_field :bgeigie_import, :by_user_name %>
+      </td>
+      <td>
+        <%= filter_field :bgeigie_import, :by_user_id %>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <b><%= t(".by_date") %></b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <%= datetime_picker :bgeigie_import, :uploaded_before %>
+      </td>
+      <td>
+        <%= datetime_picker :bgeigie_import, :uploaded_after %>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <b><%= t(".other") %></b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <%= filter_field :bgeigie_import, :q %>
+      </td>
+      <td>
+        <%= dropdown_field(:bgeigie_import, :status, [:all,:approved,:rejected,:not_approved,:not_rejected]) %>
```

Comment:
@matschaffer Do we need these all 5 conditions? Maybe 3 conditions like `:all`, `:approved`, and `:rejected`?

Reply:
Just spitballing: but `not_approved` and `not_rejected` almost seem like the should be a `not_moderated` status.

I think what the moderators are going to need is a filter to only show what has been submitted but not moderated (either approved or rejected)